misc/scripts/install-local.sh

```diff
@@ -198,10 +200,15 @@ function makeVirtualDeb {
 			unset z
 			# tab over the next line
 			echo -ne "\t"
-			ask "Do you want to install the optional dependencies" Y
-			if [[ $answer -eq 1 ]]; then
-				for optdep in "${optdeps[@]}"; do
-					deps+=" ${optdep%%: *}"
+			select_options "Select optional dependencies to install" "${#optdeps[@]}"
+			choices=( $(cat /tmp/pacstall-select-options) )
```

Comment:
Is the output newline separated in the cat command? If so I'd use `mapfile` like that's bot's saying

Reply:
It is space separated.

Reply:
Fair enough